lte/gateway/c/core/oai/test/mme_app_task/test_mme_procedures.cpp

```diff
@@ -2715,5 +2739,408 @@ TEST_F(MmeAppProcedureTest, TestTauRejDueToInvalidTac) {
   EXPECT_EQ(mme_state_p->nb_s1u_bearers, 0);
 }
 
+// Normal TAU sent in idle mode with eps bearer context status IE with inactive
+// default bearer
+TEST_F(
+    MmeAppProcedureTest, TestNormalTauReqWithEpsBearerCtxStsInactiveDefBearer) {
+  mme_app_desc_t* mme_state_p =
+      magma::lte::MmeNasStateManager::getInstance().get_state(false);
+  std::condition_variable cv;
+  std::mutex mx;
+  std::unique_lock<std::mutex> lock(mx);
+  uint8_t ebi_idx = 0;
+
+  MME_APP_EXPECT_CALLS(3, 2, 2, 1, 1, 1, 2, 3, 2, 2, 4);
```

Comment:
I see modify bearer response sent twice in the test case, but here it is expected as 3?